bot/cogs/snakes.py

```diff
@@ -40,8 +70,93 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        # await ctx.send(BeautifulSoup(text, 'lxml').find("title"))
+        name = str(name)
+        site = 'https://en.wikipedia.org/wiki/' + name
+        async with aiohttp.ClientSession() as session:
+            async with session.get(site) as resp:
+                text = await resp.text()
+                soup = BeautifulSoup(text, 'lxml')
+                title = soup.find('h1').text
+                description = soup.find('table').text
+
+                em = discord.Embed(title=title, description=description)
```

Comment:
While you should be creating the embed in this method, you should be retrieving the information over in `get_snek()`.